docs/server/generate_http_server_api_documentation.py

```diff
@@ -0,0 +1,87 @@
+""" Script to build http examples for http server api docs """
+
+import json
+import os
+import os.path
+
+from bigchaindb.common.transaction import Asset, Transaction
+
+
+TPLS = {}
+
+TPLS['post-tx-request'] = """\
+POST /transactions/ HTTP/1.1
+Host: example.com
+Content-Type: application/json
+
+%(tx)s
+"""
+
+
+TPLS['post-tx-response'] = """\
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+%(tx)s
+"""
+
+
+TPLS['get-tx-status-request'] = """\
+GET /transactions/%(txid)s/status HTTP/1.1
+Host: example.com
+
+"""
+
+
+TPLS['get-tx-status-response'] = """\
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "status": "valid"
+}
+"""
+
+
+TPLS['get-tx-request'] = """\
+GET /transactions/%(txid)s HTTP/1.1
+Host: example.com
+
+"""
+
+
+TPLS['get-tx-response'] = """\
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+%(tx)s
+"""
+
+
+def main():
+    """ Main function """
+    pub = '9RaWxppkP9UyYWA7NJb5FcgkzfJNPfvPX3FCNw2T5Pwb'
```

Comment:
Maybe use a more descriptive name for this variable (currently `pub`). Elsewhere in BigchainDB, I often see `pubkey` or `public_key`.